releng/sap-kyma-runtime/src/main/webapp/META-INF/notice.html

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+<!--
+Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the Apache License, v2.0
+which accompanies this distribution, and is available at
+http://www.apache.org/licenses/LICENSE-2.0
+  
+SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+SPDX-License-Identifier: Apache-2.0
+  -->
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1" />
+<title>Eclipse Foundation Software User Agreement</title>
```

Comment:
it should not be an eclipse user agreement

Reply:
Done